server/plugins/resolver.js

```diff
@@ -0,0 +1 @@
+module.exports = require.resolve
```

Comment:
My thinking is that for this particular use of `require.resolve`, we expect to sometimes get an error (with code `MODULE_NOT_FOUND`), rather than a correctly resolved module. In order to unit test correctly for this case, we need to stub require.resolve to return an error predictably, hence why I've separated this out to be included as a module rather than use the inbuilt `require.resolve` directly.